yarn.lock

```diff
@@ -0,0 +1,5521 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
IMO should remove this file since we're using `npm` to install dependencies

Reply:
See https://github.com/reduxjs/cra-template-redux/blob/master/package-lock.json

Reply:
Oops!